tests/functional/test-offer-fail.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+tmpfile=$(mktemp ./leaseXXXXXX)
+errfile=$(mktemp ./errXXXXXX)
+start=$(date +%Y-%m-%d)
+end=$(date -d "+5 days" +%Y-%m-%d)
+lessee_uuid=39b16f5117c24072844d55f8d20cea5d
```

Comment:
You may want to add a comment to indicate this is test1-subproject

Reply:
Ah, good point! And now that I think about it, you should now be able to specify this by name, and just do something like
lessee=test1-subproject

Reply:
Didn't realize, I'll change it to that then